keycloak.yml

```diff
@@ -0,0 +1,55 @@
+version: '3'
+
+services:
+  keycloak-postgres:
+    image: postgres:11-alpine
+    container_name: keycloak-database
+    environment:
+      POSTGRES_DB: keycloak
+      POSTGRES_USER: keycloak
+      POSTGRES_PASSWORD: keycloak
+    volumes:
+      - keycloak-postgres-data:/var/lib/postgresql/data
+    networks:
+      - keycloak-db-network
+    restart: unless-stopped
+    logging:
+      driver: journald
+      options:
+        labels: application
+        tag: keycloak-postgres
+
+  keycloak:
+    image: jboss/keycloak:4.5.0.Final
+    container_name: keycloak
+    environment:
+      DB_VENDOR: POSTGRES
+      DB_ADDR: keycloak-postgres
+      DB_DATABASE: keycloak
+      DB_USER: keycloak
+      DB_PASSWORD: keycloak
+      KEYCLOAK_USER: ${KEYCLOAK_USER:-admin}
+      KEYCLOAK_PASSWORD: ${KEYCLOAK_PASSWORD:?Please generate a secure value for KEYCLOAK_PASSWORD, using a password manager.}
+      KEYCLOAK_HOSTNAME: ${KEYCLOAK_HOSTNAME:?Please configure KEYCLOAK_HOSTNAME.}
```

Comment:
You could automate the import of the sample realm by adding:
`KEYCLOAK_IMPORT: /tmp/realm-export.json`
And adding a volume for the sample realm.
```
volumes:
- ./keycloak/realm-template.json:/tmp/realm-template.json
```

You could even replace environment variables in the real template using a startup script.

Reply:
Yes, it would be nice, but currently using the UI of Keycloak suffices.